MBBSEmu/CPU/CPUCore.cs

```diff
@@ -4075,5 +4198,43 @@ private void Flags_EvaluateSignZero(uint result)
                 Registers.F = result.IsNegative() ? Registers.F.SetFlag((ushort)EnumFlags.SF) : Registers.F.ClearFlag((ushort)EnumFlags.SF);
             }
         }
+
+        /// <summary>
+        ///     Returns the size of the given Register
+        /// </summary>
+        /// <param name="register"></param>
+        /// <returns></returns>
+        public byte GetSize(Register register)
+        {
+            return register switch
+            {
+                Register.AL => 8,
```

Comment:
As I look at these, let's change these constants to `sizeof(byte)`, `sizeof(ushort)`, `sizeof(uint)` just so people know what they are.